samples/add-features-with-contingent-values/src/main/java/com/esri/arcgismaps/sample/addfeatureswithcontingentvalues/screens/MainScreen.kt

```diff
@@ -0,0 +1,293 @@
+
+    Scaffold(
+        topBar = { SampleTopAppBar(title = sampleName) },
+        content = {
+            var showBottomSheet by remember { mutableStateOf(false) }
+            val bottomSheetState = rememberModalBottomSheetState(skipPartiallyExpanded = true)
+
+            Box {
+                MapView(
+                    modifier = Modifier
+                        .fillMaxSize()
+                        .padding(it),
+                    arcGISMap = mapViewModel.arcGISMap,
+                    graphicsOverlays = listOf(mapViewModel.graphicsOverlay),
+                    onSingleTapConfirmed = {
+                        it.mapPoint.let { point ->
+                            mapPoint = point
+                            showBottomSheet = true
+                        }
+                    }
+                )
+
+                if (showBottomSheet) {
+                    ModalBottomSheet(
+                        modifier = Modifier.wrapContentHeight(),
+                        sheetState = bottomSheetState,
+                        onDismissRequest = {
+                            showBottomSheet = false
+                            mapViewModel.clearFeature()
+                        }
+                    ) {
+                        val onBottomSheetStateChanged = { state: SheetState ->
+                            if (!state.isVisible) {
+                                showBottomSheet = false
+                                mapViewModel.clearFeature()
+                            }
+                        }
+                        BottomSheetContents(
+                            mapPoint,
+                            bottomSheetState,
+                            onBottomSheetStateChanged,
+                            featureEditState,
+                            mapViewModel::onStatusAttributeSelected,
+                            mapViewModel::onProtectionAttributeSelected,
+                            mapViewModel::onBufferSizeSelected,
+                            mapViewModel::validateContingency
+                        )
+                    }
+                }
+            }
+
+            // message dialog view model for displaying error messages
+            mapViewModel.messageDialogVM.apply {
+                if (dialogStatus) {
+                    MessageDialog(
+                        title = messageTitle,
+                        description = messageDescription,
+                        onDismissRequest = ::dismissDialog
+                    )
+                }
+            }
+        }
+    )
+}
+
+@OptIn(ExperimentalMaterial3Api::class)
+@Composable
+fun BottomSheetContents(
+    mapPoint: Point?,
+    bottomSheetState: SheetState,
+    onBottomSheetStateChange: (SheetState) -> Unit,
+    featureEditState: FeatureEditState,
+    onStatusAttributeSelect: (CodedValue) -> Unit,
+    onProtectionAttributeSelect: (CodedValue) -> Unit,
+    onBufferSizeSelect: (Int) -> Unit,
```

Comment:
```suggestion
    onBufferSizeSelected: (Int) -> Unit,
```
Also for the parameters above...